extensions/wikia/AnalyticsEngine/AnalyticsProviderGAS.php

```diff
@@ -1,6 +1,28 @@
 <?php
+
+global $wgHooks;
+
+$wgHooks['MakeGlobalVariablesScript'][] = 'AnalyticsProviderGAS::wfMakeGlobalVariablesScript';
```

Comment:
Please move hook registration to `/extensions/wikia/AnalyticsEngine/AnalyticsEngine.setup.php`